es-ds-components/components/es-data-table-simple.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div :class="tableClass">
```

Comment:
It seems like we should apply the `table-responsive` class here based on the [bootstrap docs](https://getbootstrap.com/docs/4.6/content/tables/#responsive-tables). I don't see any issue from that- were you seeing a problem with it put there?

Reply:
When I was applying `table-responsive` where we have the `table-borderless` class the width of each cell was affected, so I have now moved `table-responsive` to the the `div` tag and it works fine now. `table-borderless` works fine when applied to `div` tag or `table` class so I followed the [Bootstrap docs](https://getbootstrap.com/docs/4.6/content/tables/#borderless-table) and added it to `table` class.

Reply:
Nice!